src/main/java/Commands/RemindCommand.java

```diff
@@ -0,0 +1,89 @@
+package Commands;
+
+import Interface.*;
```

Comment:
1. Don't import everything with *
2. Interface is probably a bad name as it can cause potential conflicts in understanding (mistaken with `interface`)